ci/release/update-version.sh

```diff
@@ -29,12 +29,14 @@ function sed_runner() {
     sed -i.bak ''"$1"'' $2 && rm -f ${2}.bak
 }
 
-
-# Update version-related files using bump2version
-# (Need to execute this before other version updates because this command
-#  would checks if the git repository is dirty or not)
-./run bump_version ${NEXT_FULL_TAG}
-
 # RTD update
 sed_runner 's/version = .*/version = '"'${NEXT_SHORT_TAG}'"'/g' docs/source/conf.py
-sed_runner 's/release = .*/release = '"'${NEXT_FULL_TAG}'"'/g' docs/source/conf.py
\ No newline at end of file
+sed_runner 's/release = .*/release = '"'${NEXT_FULL_TAG}'"'/g' docs/source/conf.py
+
+sed_runner "s/${CURRENT_TAG}/${NEXT_FULL_TAG}/g" VERSION
+sed_runner "s/${CURRENT_TAG}/${NEXT_FULL_TAG}/g" python/cucim/VERSION
+sed_runner "s/${CURRENT_TAG}/${NEXT_FULL_TAG}/g" cpp/plugins/cucim.kit.cuslide/VERSION
+sed_runner "s#\[Version ${CURRENT_TAG}\](release_notes/v${CURRENT_TAG}.md)#\[Version ${NEXT_FULL_TAG}\](release_notes/v${NEXT_FULL_TAG}.md)#g" python/cucim/docs/index.md
+sed_runner "s/v${CURRENT_TAG}/v${NEXT_FULL_TAG}/g" python/cucim/docs/getting_started/index.md
+sed_runner "s#cucim.kit.cuslide@${CURRENT_TAG}.so#cucim.kit.cuslide@${NEXT_FULL_TAG}.so#g" python/cucim/docs/getting_started/index.md
+sed_runner "s#cucim.kit.cuslide@${CURRENT_TAG}.so#cucim.kit.cuslide@${NEXT_FULL_TAG}.so#g" cucim.code-workspace
```

Comment:
```suggestion
sed_runner "s/${CURRENT_LONG_TAG}/${NEXT_FULL_TAG}/g" VERSION
sed_runner "s/${CURRENT_LONG_TAG}/${NEXT_FULL_TAG}/g" python/cucim/VERSION
sed_runner "s/${CURRENT_LONG_TAG}/${NEXT_FULL_TAG}/g" cpp/plugins/cucim.kit.cuslide/VERSION
sed_runner "s#\[Version ${CURRENT_LONG_TAG}\](release_notes/v${CURRENT_LONG_TAG}.md)#\[Version ${NEXT_FULL_TAG}\](release_notes/v${NEXT_FULL_TAG}.md)#g" python/cucim/docs/index.md
sed_runner "s/v${CURRENT_LONG_TAG}/v${NEXT_FULL_TAG}/g" python/cucim/docs/getting_started/index.md
sed_runner "s#cucim.kit.cuslide@${CURRENT_LONG_TAG}.so#cucim.kit.cuslide@${NEXT_FULL_TAG}.so#g" python/cucim/docs/getting_started/index.md
sed_runner "s#cucim.kit.cuslide@${CURRENT_LONG_TAG}.so#cucim.kit.cuslide@${NEXT_FULL_TAG}.so#g" cucim.code-workspace
```

@gigony, can you merge the changes from #42 into your PR and then switch all of these to `CURRENT_TAG` variables to `CURRENT_LOG_TAG`?

Reply:
Sure. Will do.